.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # production
 /dist
+/build
```

Comment:
Old Create React App and new Vite build processes had different output folders. It's good to just exclude both options.